src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -286,6 +317,22 @@ struct xgq_cmd_cq_data_payload {
 	uint32_t resvd1;
 };
 
+/**
+ * struct xgq_cmd_cq_clk_scaling_payload: clock scaling status payload
+ *
+ * clock scaling status
+ */
+struct xgq_cmd_cq_clk_scaling_payload {
+	uint8_t has_clk_scaling:1;
+	uint8_t clk_scaling_mode:2;
+	uint8_t clk_scaling_en:1;
+	uint8_t rsvd:4;
+	uint16_t temp_shutdown_limit:8;
+	uint16_t temp_scaling_limit:8;
```

Comment:
there is a pad uint8_t. Because it is u16 aligned for this structure. The risk here is that the sizeof(struct *) might be different than we think it should be. So please add a padding.